app/javascript/components/pages/PatientEdit.test.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import Enzyme, { shallow } from "enzyme";
+import Adapter from "enzyme-adapter-react-16";
+import PatientEdit from "./PatientEdit";
+
+const mockPatient = { first_name: "Stefani", last_name: "F" };
+
+global.fetch = () =>
+  Promise.resolve({ json: () => Promise.resolve(mockPatient) });
+
+Enzyme.configure({ adapter: new Adapter() });
+
+describe("when PatientEdit renders", () => {
+  it("has a button to edit the patient info", () => {
+    const info = shallow(<PatientEdit id={1} />);
+    let infoButton = info.find('Button[children="Edit Patient"]')
```

Comment:
prefer `const` to let here. `let` is mutable and there's no mutation going on here.

Reply:
@archae0pteryx can do! will change `let`s to `const`